src/content/guides/getting-started.md

```diff
@@ -335,4 +336,6 @@ webpack-demo
 
 T> If you're using npm 5, you'll probably also see a `package-lock.json` file in your directory.
 
+W> Do not compile untrusted code with webpack. It can potentially allow a bad performer to execute malicious code on your machine, your CI, or in the Web browser of the user.
```

Comment:
```suggestion
W> Do not compile untrusted code with webpack. It could lead to execution of malicious code on your computer, remote servers, or in the Web browsers of the end users of your application.
```